python/nano/src/bigdl/nano/deps/neural_compressor/pytorch/quantized_model.py

```diff
@@ -52,12 +53,20 @@ def _load(path, model, example_inputs=None):
             model is not None,
             errMsg="FP32 model is required to create a quantized model."
         )
+        ipex_quantization = False
+        # only ipex quantization saves this file
+        if os.path.exists(os.path.join(path, "best_configure.json")):
+            ipex_quantization = True
+        if ipex_quantization:
+            invalidInputError(example_inputs is not None,
```

Comment:
why this parameter is named `example_inputs`, while others are all `input_sample`

Reply:
it's just an internal paramater and it's because INC use `example_inputs`, so I think keep it like this is ok.